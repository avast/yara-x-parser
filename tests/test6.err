SyntaxError("expected a name", 38..38)
SyntaxError("expected meta, strings or condition keyword", 92..92)
SyntaxError("expected meta, strings or condition keyword", 98..98)
SyntaxError("expected meta, strings or condition keyword", 102..102)
SyntaxError("expected meta, strings or condition keyword", 104..104)
SyntaxError("expected meta, strings or condition keyword", 106..106)
SyntaxError("expected meta, strings or condition keyword", 114..114)
SyntaxError("expected meta, strings or condition keyword", 117..117)
SyntaxError("expected meta, strings or condition keyword", 119..119)
SyntaxError("expected meta, strings or condition keyword", 141..141)
SyntaxError("expected meta, strings or condition keyword", 147..147)
SyntaxError("expected meta, strings or condition keyword", 150..150)
SyntaxError("Invalid character", 98..99)