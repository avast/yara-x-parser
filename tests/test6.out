SOURCE_FILE@0..157
  COMMENT@0..16 "//Global comment"
  WHITESPACE@16..18 "\n\n"
  RULE@18..156
    COMMENT@18..32 "//Rule comment"
    WHITESPACE@32..33 "\n"
    RULE_KW@33..37 "rule"
    WHITESPACE@37..38 " "
    ERROR@38..47
      CONDITION_KW@38..47 "condition"
    WHITESPACE@47..48 "\n"
    BLOCK_EXPR@48..156
      L_BRACE@48..49 "{"
      WHITESPACE@49..51 "\n\t"
      COMMENT@51..71 "//Rule block comment"
      WHITESPACE@71..74 "\n\n\t"
      COMMENT@74..90 "//String comment"
      WHITESPACE@90..92 "\n\t"
      ERROR@92..98
        IDENTIFIER@92..98 "string"
      ERROR@98..99
        ERROR@98..99 "*"
      WHITESPACE@99..102 "\n\t\t"
      ERROR@102..103
        IDENTIFIER@102..103 "a"
      WHITESPACE@103..104 " "
      ERROR@104..105
        ASSIGN@104..105 "="
      WHITESPACE@105..106 " "
      ERROR@106..111
        INT_LIT@106..111 "00000"
      WHITESPACE@111..114 "\n\t\t"
      ERROR@114..116
        VARIABLE@114..116 "$b"
      WHITESPACE@116..117 " "
      ERROR@117..118
        ASSIGN@117..118 "="
      WHITESPACE@118..119 " "
      ERROR@119..124
        STRING_LIT@119..124 "\"bar\""
      WHITESPACE@124..126 "\n\t"
      CONDITION@126..153
        CONDITION_KW@126..135 "condition"
        COLON@135..136 ":"
        WHITESPACE@136..139 "\n\t\t"
        ERROR@139..140
          IDENTIFIER@139..140 "a"
        WHITESPACE@140..141 " "
        ERROR@141..144
          IDENTIFIER@141..144 "ord"
        WHITESPACE@144..147 "\n\t\t"
        EXPRESSION_STMT@147..149
          LITERAL@147..149
            VARIABLE@147..149 "$b"
        WHITESPACE@149..150 " "
        ERROR@150..153
          IDENTIFIER@150..153 "ant"
      WHITESPACE@153..155 " \n"
      R_BRACE@155..156 "}"
  WHITESPACE@156..157 "\n"
